climada/hazard/trop_cyclone/__init__.py

```diff
@@ -0,0 +1,5 @@
+from climada.hazard.trop_cyclone.trop_cyclone import *
+from climada.hazard.trop_cyclone.trop_cyclone_windfields import compute_windfields_sparse, compute_angular_windspeeds, tctrack_to_si, \
+    get_close_centroids, KN_TO_MS, KM_TO_M, KM_TO_M, H_TO_S, NM_TO_KM, KMH_TO_MS, MBAR_TO_PA, \
+    DEF_MAX_DIST_EYE_KM, DEF_INTENSITY_THRES, DEF_MAX_MEMORY_GB, MODEL_VANG, DEF_RHO_AIR, DEF_GRADIENT_TO_SURFACE_WINDS, \
+    T_ICE_K, V_ANG_EARTH
```

Comment:
Does it make sense to "expose" all of these constants here? Are they, or can they be, used in other contexts at all? 🤔

Reply:
The constants are not private and were exposed before using the * import in the init file of hazard. So we basically allowed this usage before, and it happened e.g. in petals: https://github.com/CLIMADA-project/climada_petals/blob/main/climada_petals/hazard/tc_tracks_forecast.py#L50. I just expose them the same as before (except for the private members, which I no longer expose now).

Do you have another suggestion?

Reply:
Alright, that's fine! I was simply not aware of this use case. Let's keep exposing them for backwards compatibility.